core/src/banking_stage.rs

```diff
@@ -584,33 +572,34 @@ impl BankingStage {
     }
 
     fn spawn_vote_worker(
-        id: u32,
-        packet_receiver: BankingPacketReceiver,
+        tpu_receiver: BankingPacketReceiver,
+        gossip_receiver: BankingPacketReceiver,
         decision_maker: DecisionMaker,
         bank_forks: Arc<RwLock<BankForks>>,
         committer: Committer,
         transaction_recorder: TransactionRecorder,
         log_messages_bytes_limit: Option<usize>,
         vote_storage: VoteStorage,
     ) -> JoinHandle<()> {
-        let packet_receiver = PacketReceiver::new(id, packet_receiver);
+        let tpu_receiver = PacketReceiver::new(1, tpu_receiver);
+        let gossip_receiver = PacketReceiver::new(0, gossip_receiver);
```

Comment:
Can give `0` to both `PacketReceiver`s here, so to be consistent with `QosService` below - that `0` is for vote.

Reply:
Let's just kill `id` - it's for a debug log that nobody cares about.